java/ql/src/change-notes/2024-05-01-remove-local-query-variants.md

```diff
@@ -0,0 +1,4 @@
+---
+category: majorAnalysis
```

Comment:
Should this be `breaking`? Anyone referencing the local variants most likely will have their workflows broken.

Reply:
Yes that makes sense!